### DISCUSSION TITLE
Trigger error without submitting

### DISCUSSION THREAD

**Discussion by vicasas** [react-hook-form/react-hook-form#3891] [Q&A]
I am doing a Step form component with MUI but I need to trigger the errors without submitting when I go to the next step, is there any way to do this with RHF? In the documentation I didn't find much about the errors

https://react-hook-form.com/en/api/#errors

**Comment by bluebill1049** [comment]
`trigger` api?

**Reply by vicasas** [reply]
I had not paid attention to that api, I will review it, Thanks :D

**Reply by bluebill1049** [reply]
https://react-hook-form.com/api#trigger
